lib/config.js

```diff
@@ -74,6 +74,9 @@ module.exports = (function () {
           enabled: false,
           clientId: "replace me with the real value",
           clientSecret: "replace me with the real value",
+          authorizationURL: "leave me empty to https://github.com/login/oauth/authorize",
+          tokenURL: "leave me empty to https://github.com/login/oauth/access_token",
+          userProfileURL: "leave me empty to https://api.github.com/user",
           redirectURL: ""
```

Comment:
There is also the concern of people using the github authentication, updating Jingo after we merge this PR, not updating their config file, and then _probably_ failing the authentication because the default values of those fields are not a valid value.